### ISSUE TITLE
[10_W0] Trigger user-response messages in language selected by the user

### ISSUE THREAD

**Issue by tarunima** [tattle-made/DAU#100]
### Scope of Work
- Take new templates in non-english languages from DAU secratariat
- Whitelist them using gupshup unify console
- Add it to codebase

**Comment by dennyabrain** [comment]
Since we have done this for the english templates, it would be relatively simpler to replicate this for the other languages. 
Most of the code relevant to this feature is [here](https://github.com/tattle-made/dau-dashboard/tree/main/test/dau/user_message/templates). And their associated templates are [here](https://github.com/tattle-made/dau-dashboard/tree/main/test/dau/user_message/templates)

The static templates are stored [here](https://github.com/tattle-made/dau-dashboard/tree/main/priv/static/gupshup_templates)

A foot gun I had encountered last time was that when I was editing these template files using VSCode, it was removing certain spaces at the end of line which would cause the template to not match the text whitelisted on gupshup. I could not figure out which vscode extension was doing it, so I resorted to using vim to edit the files as a stopgap solution. Its a bit inconvinient but flagging this in case you end up encountering the same issue.
